lib/methods/dynamics/webApi/read.js

```diff
@@ -29,6 +29,21 @@ module.exports = (
     mappings
   } = internals.dynamics
 
+  /**
+   * Break a query into an array of smaller queries
+   * @param {Object} query generally in like { accountIds: [nnn] }
+   * @param {String} fieldName the name of the field containing the ids
+   * @returns {Array} an array of queries like [ {accountIds:[ ]}, {accountIds:[ ]} ]
+   */
+  const chunkQuery = (query, fieldName) => {
+    const splitIds = _.chunk(query[fieldName], 100) // TODO: how many ids can a GET filter querystring handle?
```

Comment:
should IDs also be in config?

Reply:
but this the plugin and it doesn't have a config :(